README.md

```diff
@@ -131,63 +131,103 @@ cmake .. -DCMAKE_CXX_FLAGS="-O3 -g" \
 
 ## RUNNING SIMULATION:
 
-Note that the CoreNEURON simulator dependends on NEURON to build the network model: see [NEURON](https://www.neuron.yale.edu/neuron/) documentation for more information. Once you build the model using NEURON, you can launch CoreNEURON on the same or different machine by:
+Note that the CoreNEURON simulator depends on NEURON to build the network model: see [NEURON](https://www.neuron.yale.edu/neuron/) documentation for more information. Once you build the model using NEURON, you can launch CoreNEURON on the same or different machine by:
 
 ```bash
 export OMP_NUM_THREADS=2     #set appropriate value
-mpiexec -np 2 build/apps/coreneuron_exec -e 10 -d /path/to/model/built/by/neuron -mpi
+mpiexec -np 2 build/apps/coreneuron_exec -e 10 --mpi input -d /path/to/model/built/by/neuron
 ```
 
 [This tutorial](https://github.com/nrnhines/ringtest) provide more information for parallel runs and performance comparison.
 
+### Command Line Interface
+
+:warning: :warning: :warning: **In a recent update the command line interface was updated, so please update your scripts accordingly!**
+
+Some details on the new interface:
+
+The new command line interface is based on CLI11. All the previous options are still supported but they are organized differently. You can find more details by running `coreneuron_exec --help-all`.
+
+Multiple character options with single dash (e.g. `-gpu`) are not supported anymore. All multiple characters options now require a double dash (e.g. `--gpu`), but single characters option still support a single dash (e.g. `-g`).
+
+The structure of a command is the following: `./apps/coreneuron_exec [OPTIONS] [SUBCOMMAND]`. The `OPTIONS` should **always** be written before any of the `SUBCOMMAND`. To know which of the parameter is an `OPTION` or a `SUBCOMMAND` run `coreneuron_exec --help-all`.
+
+To access a parameter that falls under any of the subcommands category it is necessary to include in the command line a certain keyworld first. For example to use the `-d` option, one must enter in the `input` subcommands first by using the `input` keyword first:
```

Comment:
```suggestion
To access a parameter that falls under any of the subcommands category it is necessary to include in the command line a certain keyword first. For example to use the `-d` option, one must enter in the `input` subcommands first by using the `input` keyword first:
```

Reply:
Also, the paragraph should be updated for clarity. It's not really clear, in this form.

Reply:
By including the definition of `options` as parameters that do not belong to any of the `subcommands` it should be a bit clearer. But I understand it can be a bit foggy. What parts in particular would you improve? What things are not clear for you?